src/io/sampling.jl

```diff
@@ -350,6 +350,40 @@ function fix_factor!(d::Domain; factors...)::Nothing
     end
 end
 
+"""
+    fix_factor_bounds!(d::Domain, factor::Symbol, new_bnds::Tuple)
+    fix_factor_bounds!(d::Domain; factors...)
+
+Fix bounds of a parameter for sampling to those provided.
+
+Note: Changes are permanent. To reset, either specify the original value(s)
+      or reload the Domain.
+
+# Examples
+```julia
+# Fix `wave_stress` to specified bounds
+fix_factor_bounds!(dom, :wave_stress, (0.1,0.2))
+```
+"""
+function fix_factor_bounds!(d::Domain, factor::Symbol, new_bnds::Tuple)::Nothing
```

Comment:
I think I went with fix because of the other fix_factor function, but you're right, "set" makes more sense

It works with non-uniform but you'd have to know to add 3 parameters in the tuple for triangular so I'll add a check that the number of args is correct